test/unit/assertions/satisfyApiSpec.test.js

```diff
@@ -33,8 +33,12 @@ const { expect } = chai;
 
 for (const spec of openApiSpecs) {
   const { openApiVersion, pathToApiSpec } = spec;
+
+  // add a path to a server if testing in openAPI 3
+  const serverPath = spec.openApiVersion == 3 ? '/local' : '';
```

Comment:
nit: `===`. Did our linter not pick this up?

Reply:
I don't believe that it did, I'll check the settings for this too.

Reply:
Since we define a different `.eslintrc` for the tests, I think the base rules no longer apply - this could be something we should fix